src/scripts/main.js

```diff
@@ -1,5 +1,33 @@
 'use strict';
 
-document.addEventListener('click', e => {
-  // write code here
+document.addEventListener('click', el => {
+  const spider = document.querySelector('.spider');
+  const wall = document.querySelector('.wall');
+  const borderWidth = getComputedStyle(wall).borderLeftWidth;
+  const leftSide = wall.getBoundingClientRect().left;
+  const rightSide = wall.getBoundingClientRect().right;
+  const topSide = wall.getBoundingClientRect().top;
+  const bottomSide = wall.getBoundingClientRect().bottom;
+
```

Comment:
These variables are created every time you click on the wall. You can create them globally so that you create them only once, and then can reuse them on every click.